examples/jsm/nodes/materials/Materials.js

```diff
@@ -32,19 +32,21 @@ NodeMaterial.fromMaterial = function ( material ) {
 
 	if ( materialLib[ type ] === undefined ) {
 
-		return material; // is already a node material or cannot be converted
+		if ( material.isNodeMaterial !== true ) {
 
-	}
+			throw new Error( `NodeMaterial: Material "${ material.type }" is not compatible.` );
 
-	const nodeMaterial = new materialLib[ type ]( material );
+		}
 
-	for ( const key in material ) {
+		return material; // is already a node material
 
-		if ( nodeMaterial[ key ] === undefined ) {
+	}
 
-			nodeMaterial[ key ] = material[ key ]; // currently this is needed only for material.alphaTest
+	const nodeMaterial = new materialLib[ type ]();
```

Comment:
I'm not sure about this... `new materialLib[ type ]( material )` internally uses `Material.setValues( material )`, which copies values not only by simply assigning, but also via `.copy()`, but also produces a warning if there is not a key in new material (`NodeMaterial`).
Maybe we can add an option to that function to disable warnings? /cc @mrdoob @Mugen87

Reply:
Or maybe we can add `_listeners` as an exception in `Material.setValues`.

Reply:
Hmm... The point is that `.fromMaterial()` can be used outside of `WebGPURenderer`, but for `WebGPURenderer` is good not copy/clone to not add unnecessary duplicity in memory.